tests/test_eval.py

```diff
@@ -49,19 +54,38 @@ def evaluator_ensemble_mobilenet():
              'target_size': [224, 224, 3]
              }
 
-    with open(os.path.abspath('tmp/fixtures/models/mobilenet_1/model_spec.json'), 'w') as outfile:
+    with open(os.path.abspath('tmp/fixtures/models/ensemble/mobilenet_1/model_spec.json'), 'w') as outfile:
         json.dump(specs, outfile)
 
-    with open(os.path.abspath('tmp/fixtures/models/mobilenet_2/model_spec.json'), 'w') as outfile:
+    with open(os.path.abspath('tmp/fixtures/models/ensemble/mobilenet_2/model_spec.json'), 'w') as outfile:
         json.dump(specs, outfile)
 
     return Evaluator(
-        ensemble_models_dir='tmp/fixtures/models/',
+        ensemble_models_dir='tmp/fixtures/models/ensemble/',
         combination_mode='arithmetic',
         batch_size=1
     )
 
 
+@pytest.fixture('function')
+def evaluator_mobilenet_class_combine():
```

Comment:
If we're going to start using the "inference" naming, may we change all the "combine" words with "inference"? 🤔